metadata.yaml

```diff
@@ -24,6 +24,8 @@ containers:
 provides:
   grafana-source:
     interface: grafana_datasource
+  remote-write-server:
```

Comment:
Nitpick: Is `remote-write-server` the agreed upon relation name ? I was wondering what does `remote-write-sink` sound like, keeping in with the tradition established by `grafana-source` ?

Reply:
The idea here is that at some point Prometheus might both require and provide this interface, at which point we would add remote-write-client. I think that is the most clear way to do it but I am happy to change it to source and sink if you would like.

Reply:
On the one hand, `grafana-source` has only stayed stable by luck of the draw rather than any well-thought-out naming 😉

On the other hand, if this is Prometheus only, I’d make it very explicit and just call it `promtail `

Reply:
Please do check with @mmanciop what his preferences here are.

Reply:
I think the intuition is good, but let’s call it `receive-remote-write`

Reply:
done